doc/pages/faces.asciidoc

```diff
@@ -2,183 +2,188 @@
+    else onto whatever color happens to be used at the moment. For technical
+    reasons, alpha values must be >16.
+
+attributes::
+    A string whose individual letters each set an attribute:
         *u*:::
-            underline
+            Underline
         *c*:::
-            curly underline, takes precedence over underline if both are
-            specified
+            Curly underline
+            Note: This takes precedence over underline if both are specified.
         *r*:::
-            reverse
+            Reverse
         *b*:::
-            bold
+            Bold
         *B*:::
-            blink
+            Blink
         *d*:::
-            dim
+            Dim
         *i*:::
-            italic
+            Italic
         *s*:::
-            strikethrough
+            Strikethrough
         *F*:::
-            final, override the previous face instead of merging with it,
-            and this face will only be replaced if another face with
-            the final attribute is applied
+            Final::::
+                Override the previous face instead of merging with
+                it. Can only be replaced if another face with the final
+                attribute is applied.
         *f*:::
-            final foreground, as final but only applies to face's
-            foreground color
+            Final foreground::::
+                Same as final, but only applies to a face's foreground color.
         *g*:::
-            final background, as final but only applies to face's
-            background color
+            Final background::::
+                Same as final, but only applies to a face's background color.
         *a*:::
-            final attributes, as final but only applies to face's
-            attributes
+            Final attributes::::
+                Same as final, but only applies to a face's attributes.
 
-'base'::
-    The base face on which this face applies, which can be any face name,
-    as long as no cycle is introduced. A face can reference itself, in
-    which case it will apply on top of the parent scope version.
+base::
+    The face onto which the rest of the face applies. Its value can be any
+    face name, as long as no cycle is introduced. A face can reference itself,
+    in which case it will apply on top of the parent scope.
 
-== Builtin faces
+== Built-in faces
 
-The following default faces are used by color schemes to highlight certain
-areas of the user interface:
+The following faces are used by color schemes to highlight certain areas of
+the user interface:
 
 *Default*::
-    default colors
+    Default colors:::
+        The default foreground and background colors. If the value of *default*
+        is used here, then the colors used will be your terminal's defaults.
 
 *PrimarySelection*::
-    main selection face for every selected character except the cursor
+    Main selection face for every selected character except the cursor.
```

Comment:
I agree with adding capitalization/punctuation to full sentences but this is an ellipsis, where we usually don't do either.
There's a similar style in `doc expansions`, `doc commands` and docstrings in autoinfo (`:echo`). They are "definition lists" where the verb is left out, because it's implied by the context

Reply:
The ellipsis is the omitted part. It's still a sentence - an elliptical sentence. Regardless, as mentioned in the other comment, these definitions don't need capitalization or punctuation at the end. I'll revert these changes.

Reply:
I see now the original issue. The problem is that some definitions require more than one clause to explain. Writing such clauses is messy without the normal delimiters of capitalization on the left end and punctuation on the right end. For example, the definition of the `final` attribute:

> Override the previous face instead of merging with it. Can only be replaced if another face with the final attribute is applied.

If such multi-clause definitions are required, I would think it best to adopt a uniform look for all definitions, regardless of the number clauses, if any.

The alternative would be to begin and end certain definitions one way, and other definitions a different way, which lacks uniformity and hence is jarring.

Reply:
You're right, thanks for your tenacity. I agree that we should achieve consistency.

How about a compromise: keep the unabbreviated attribute lowercase and put everything beyond that in proper sentences. Rough sketch:

```diff
        *F*:::
            final::::
                Override the previous face instead of merging with
                it. Can only be replaced if another face with the final
                attribute is applied.

        *c*:::
            curly underline::::
                Note: This takes precedence over underline if both are specified.
```

Reply:
Okay, will do.

P.S. I suspect it's more stubbornness than tenacity...